src/node.cc

```diff
@@ -994,6 +984,16 @@ InitializationResult InitializeOncePerProcess(int argc, char** argv) {
     }
   }
 
+  if (per_process::cli_options->use_largepages == 1 ||
+      per_process::cli_options->use_largepages == 2) {
+    if (node::IsLargePagesEnabled()) {
```

Comment:
Shouldn't we separately report that largepages are unsupported on this platform?